src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-appx/xunitrunner-appx.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env py
```

Comment:
We shouldn't need this runner script anymore correct?  My understanding is all of this logic belongs with the product cmd file.